src/main.rs

```diff
@@ -1,28 +1,30 @@
 //! The main entry point for the `muse2` command-line tool.
 use ::log::info;
-use clap::{Arg, Command};
+use clap::{Parser, Subcommand};
 use muse2::log;
 use muse2::model::Model;
 use muse2::settings::Settings;
 use std::path::PathBuf;
 
-fn build_run_command() -> Command {
-    Command::new("run")
-        .about("Use 'muse2 run <MODEL_DIR>' to run a simulation")
-        .arg(
-            Arg::new("MODEL_DIR")
-                .help("Path to the model directory")
-                .required(true)
-                .index(1)
-                .value_parser(clap::value_parser!(PathBuf)),
-        )
+#[derive(Parser)]
+#[command(name = clap::crate_name!())]
+#[command(version = clap::crate_version!())]
+#[command(about = clap::crate_description!())]
```

Comment:
It looks like `clap` automatically gets these values. so this can simply be:
```suggestion
#[command(version, about)]
```

(looks like `name` can be omitted entirely and it gets the crate name)

Reply:
Much cleaner!